README.md

```diff
@@ -49,13 +43,12 @@ Configuring Your AndroidManifest
     </application>
     ```
 
--   Enable the `INTERNET`, `ACCESS_NETWORK_STATE` and `GET_TASKS` permissions:
+-   Optional: Enable the `ACCESS_NETWORK_STATE` and `GET_TASKS` permissions:
```

Comment:
I agree with Jake, these should not be automatically added (only INTERNET should be automatic, as it is 100% required).